paladin/mod.ts

```diff
@@ -0,0 +1,186 @@
+import { Drash } from "../deps.ts";
+
+type ReferrerPolicy =
+  "" |
+  "no-referrer" |
+  "no-referrer-when-downgrade" |
+  "same-origin" |
+  "origin" |
+  "strict-origin" |
+  "origin-when-cross-origin" |
+  "strict-origin-when-cross-origin" |
+  "unsafe-url"
+
+interface Configs {
+  "X-XSS-Protection"?: boolean
+  "Referrer-Policy"?: ReferrerPolicy,
+  "X-Content-Type-Options"?: boolean
+  hsts?: {
+    maxAge?: boolean | number,
+    includeSubDomains?: boolean,
+    preload?: boolean
+  },
+  "X-Powered-By"?: boolean | string
+  "X-Frame-Options"?: "DENY" | "SAMEORIGIN" | boolean | string, // eg ALLOW-FROM www.example.com
+  expectCt?: {
+    enforce?: boolean,
+    maxAge?: number,
+    reportUri?: string
+  },
+  "X-DNS-Prefetch-Control"?: boolean,
+  "Content-Security-Policy"?: string
+}
+
+/**
+ * A logger middleware inspired by https://www.npmjs.com/package/morgan.
+ *
+ * @param configs - See IDexterConfigs
+ */
+export function Paladin(
+  configs?: Configs,
```

Comment:
i noticed a lot of `!` below. what if we do `configs: Configs = {}`. would that help keeping the usage of `!` low?

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/8d64129735f60dc332d67851d46630279059607d